README.md

```diff
@@ -35,48 +88,228 @@ Here are examples of current capabilities:
 ##### Response:
 ```json
 {
-  "coming": "soon"
+  "id":"2",
+  "title":"HubKick",
+  "tag":"Send players to lobby on kick. \/lobby \/ hub",
+  "current_version":"1.7.1",
+  "native_minecraft_version":null,
+  "supported_minecraft_versions":[
+    "1.7",
+    "1.8",
+    "1.9"
+  ],
+  "icon_link":"https:\/\/www.spigotmc.org\/data\/resource_icons\/0\/2.jpg?1482076430",
+  "author":{
+    "id":"106",
+    "username":"LaxWasHere"
+  },
+  "premium":{
+    "price":"0.00",
+    "currency":""
+  },
+  "stats":{
+    "downloads":"7719",
+    "updates":"22",
+    "reviews":{
+      "unique":"11",
+      "total":"11"
+    },
+    "rating":"5"
+  },
+  "description":"[CENTER][I][B][URL='https:\/\/discord.gg\/SvFEBGW'][IMG]https:\/\/i.imgur.com\/inmwaK3.png[\/IMG] [\/URL]| [URL='https:\/\/www.paypal.me\/LaxWasHere'][IMG]https:\/\/i.imgur.com\/h5YgdLC.png[\/IMG][\/URL] | [URL='https:\/\/github.com\/AwesomePowered\/HubKick'][IMG]https:\/\/i.imgur.com\/ikjLwXh.png[\/IMG][\/URL][\/B][\/I][\/CENTER]\n[I][B]Info[\/B]\nWhen a player get's kicked from the server, this plugin will forward him\/her to the server you specified in the config.yml.\nKick everyone and shutdown the server or just kick everyone.\n\n[B]Configuration[\/B]\n[code]#Hub is the server you want the players to be sent to.\nHubServer: hub\n\n#Enable this and it will pick a hub on random when kicked\nHubs:\n- Hub\n- Lobby\nrandomHubs: false\n\n#Regex pattern for forcekicking a player. No touchy touchy pls\nignorePattern: '(\\bafk|AFK|-f\\b)'\n\n#Prefix\nprefix: '&4[&aHubKick&4]'\n\n#Message to be sent when kicking everyone\/ shutting down the server.\nKickallMessage: '&a&lServer shutting down, you have been kicked to the hub!'\n\n#Should the plugin send the users on hub on kick?\nHubOnKick: true\n[\/code]\n\n[B]Installation[\/B]\n [\/I]\n[LIST]\n[I]\n[LIST][*]Drop the HubKick.jar at your \/plugins folder[\/LIST]\n[LIST][*]Restart Server[\/LIST]\n[LIST][*]Get Kicked[\/LIST][\/I]\n[\/LIST]\n[I][B]Commands:[\/B]\n[CODE]\/hub (sends you to he hub server)\n\/lobby (same as above)\n\/alltolobby (sends everyone to lobby)\n\/lobbyall (Same as above)\n\/sendplayer(Sends player to a server)\n\/sendp(^)\n\/shutdown (sends everyone to lobby and shuts down the server)\n\/forcekick (kicked the player off the network) (or simply add -f to a kick message)\n\/fkick (same as above)\n[\/CODE]\n\n[B]Permissions:[\/B]\n [\/I]\n[LIST]\n[I]\n[LIST][*]hubkick.command[\/LIST]\n[LIST][*]hubkick.kickall[\/LIST]\n[LIST][*]hubkick.send[\/LIST]\n[LIST][*]hubkick.shutdown[\/LIST]\n[LIST][*]hubkick.forcekick[\/LIST][\/I]\n[\/LIST]\n[I][URL='https:\/\/github.com\/AwesomePowered\/HubKick']Github[\/URL][\/I]"
 }
 ```
 
 #### getResourcesByAuthor
 ##### Parameters: `id`, required, the author id
```

Comment:
Can you please add that there is a page parameter here (same as listResources)? I forgot to do so.